FAQ.md

```diff
@@ -0,0 +1,28 @@
+#### How to run visual tests
```

Comment:
Why add another file? I would prefer to keep these things in fewer files.

Reply:
I was not sure where to put that. And there 3 PR from the past two weeks that had an issue with "how to fix failing visual tests". There could be more but that is the one that I found this week

Reply:
I'm ok and to remove it .. maybe the only suitable place to move it will be into BUILD.md

Reply:
Its fine then for now, we can always update it later.